homeassistant/components/homematicip_cloud/binary_sensor.py

```diff
@@ -136,6 +136,44 @@ async def async_setup_entry(
         async_add_entities(entities)
 
 
```

Comment:
wonder if device_class of `connectivity` is suitable here. it does have slightly different icons though. can be done in new PR if desirable.

https://github.com/home-assistant/frontend/blob/5ff52ea1137c6d8640d6e6979686441940031966/src/common/entity/binary_sensor_icon.ts#L15